lib/boilerPlate.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const util = require('util');
+
+/**
+ *
+ */
+class BoilerPlate extends Error{
+    /**
+     * [setMessage description]
+     * @return {[type]} [description]
+     */
+    setMessage(variables){
+        this.name = 'BoilerPlate';
+        this.message = `insert your message and your ${util.inspect(variables, {depth: null, colors: true})} in here!`;
+        return this.message;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/RIAEvangelist/node-error-classes/file/1939102910/issues/source?bid=3073169&fileBranchId=3104106#l15)